ontobio/rdfgen/gocamgen/collapsed_assoc.py

```diff
@@ -66,7 +66,10 @@ def collapse_annotations(self, associations: List[GoAssociation]):
             for wf in separated_with_froms:
                 ca = self.find_or_create_collapsed_association(a, wf)
                 association_line = CollapsedAssociationLine(a, wf.line)
+                # adding binding annotation with/from to the evidence line as well
+                association_line2 = CollapsedAssociationLine(a, ca.with_from())
                 ca.lines.append(association_line)
+                ca.lines.append(association_line2)
```

Comment:
I think this is creating a duplicate evidence individual, based on the test model I examined:
![image](https://user-images.githubusercontent.com/2678599/119587755-2f30eb80-bd84-11eb-92ee-5ab5a5014183.png)
Instead of making the extra `CollapsedAssociationLine` here, I think we can try tweaking the `add_to_header` vs `add_to_line` conditional below in `get_with_froms` (see other comment).

Reply:
Grr! Dang github won't let me comment outside of changed code. I'll try this:
```
if wf_entity and wf_entity.get("taxon") == subject_entity["taxon"]:
    wf_separated.add_to_header(wf_id_str)
else:
    wf_separated.add_to_line(wf_id_str)
```
Header is basically what's getting converted to a `has_input`. Line is the "with/from" that shows up on the evidence. Without testing, I'm gonna assume, if we just want to always keep the "with/from" on the evidence, we just need to remove the `else:` so that `wf_separated.add_to_line(wf_id_str)` always runs. Like:
```
if wf_entity and wf_entity.get("taxon") == subject_entity["taxon"]:
    wf_separated.add_to_header(wf_id_str)
wf_separated.add_to_line(wf_id_str)
```
@sierra-moxon We can quick-confirm a model in my test Noctua tomorrow.